Controls/MhwDataGrid.xaml.cs

```diff
@@ -65,6 +69,85 @@ protected MhwDataGrid() {
         protected abstract void On_Sorting(object sender, DataGridSortingEventArgs e);
 
         protected abstract void On_Cell_MouseClick(object sender, MouseButtonEventArgs e);
+
+        private void DataGrid_KeyDown(object sender, KeyEventArgs e)
+        {
+            try
+            {
+                if (sender is DataGrid MyDataGrid)
+                {
+                    if ((Keyboard.Modifiers & ModifierKeys.Control) == ModifierKeys.Control && e.Key == Key.V)
+                    {
+                        // Gérer le collage des données du presse-papiers
```

Comment:
Please translate all comments into Eng. I'm grateful you're documented things, but I'd rather keep the lang of all comments consistent which means Eng.